csvcubed/tests/unit/cli/inspect/test_metadataprocessor.py

```diff
@@ -20,6 +20,36 @@ def test_metadata_dataset_json_ld_to_rdf_loading():
     assert any(csvw_metadata_rdf_graph)
 
 
+def test_metadata_dataset_json_ld_to_rdf_loading_when_path_contains_url_encodable_chars():
+    """
+    Metadata dataset RDF graph should not be None when the path contains url encodable chars.
+    """
+    dir_path = _test_case_base_dir / "url_enc@dable_char_@2path"
+    csvw_metadata_json_path = dir_path / "alcohol-bulletin.csv-metadata.json"
+
+    metadata_processor = MetadataProcessor(csvw_metadata_json_path)
+    csvw_metadata_rdf_graph = metadata_processor.load_json_ld_to_rdflib_graph()
+
+    assert csvw_metadata_rdf_graph is not None
+    assert any(csvw_metadata_rdf_graph)
+
+    assert (
+        URIRef(
+            f"file://{dir_path}/alcohol-bulletin.csv#component/alcohol-sub-type",
+        ),
+        None,
+        None,
+    ) in csvw_metadata_rdf_graph
+
+    assert (
+        URIRef(
+            f"file://url_enc%40dable_char_%402path/alcohol-bulletin.csv#component/alcohol-sub-type",
```

Comment:
I know you're asserting that this doesn't exist, but this should have `{dir_path}` in it too.